### PR TITLE
Validate stack parameters against CloudFormation validation

### PR DESCRIPTION
**Pull request by atward**

### CODE REVIEW

File: bespin/option_spec/stack_objs.py [hunk 1]
```diff
@@ -22,6 +22,12 @@
 import os
 import re
 
+# python 2.6 set compat
+try:
+    set
+except NameError:
+    from sets import Set as set
```
Comment (delfick) [inline bespin/option_spec/stack_objs.py]: I'm pretty sure 2.6 has set.

2.6.9 certainly does (that's the 2.6 installed on my laptop)

**Review by delfick — APPROVED** [review]
Looks fine to me

File: bespin/option_spec/stack_objs.py (hunk 1 above)
Comment (atward) [reply bespin/option_spec/stack_objs.py]: Confirmed, think I must be confusing with 2.3-2.4 compat
https://docs.python.org/2.6/library/stdtypes.html#set

**Comment by delfick** [comment]
:+1:

**Pull request merged**